src/request/billing.spec.ts

```diff
@@ -0,0 +1,11 @@
+import Billing from './billing';
+
```

Comment:
Do you think it'd be beneficial to write a test that ensures that `Billing` is an extension of the `Location` class. I can see it being somewhat of an integration test.